improver_tests/acceptance/test_generate_realizations.py

```diff
@@ -113,6 +113,30 @@ def test_realizations(tmp_path):
     acc.compare(output_path, input_path)
 
 
+@pytest.mark.slow
+def test_ecc_bounds_warning(tmp_path):
+    """
+    Test use of ECC to convert one set of percentiles to another set of
+    percentiles, and then rebadge the percentiles to be ensemble realizations.
+    Data in this input exceeds the ECC bounds and so tests ecc_bounds_warning
+    functionality.
+    """
```

Comment:
Is there a way to check the warning is raised?  Eg I think there is a `with pytest.warns` or something analagous available.

Reply:
Yep. Done.

Reply:
Sorry I should have also commented on the other test file - I think that one merits a `pytest.warns` as well.

Reply:
I've added a check that a warning is raised using `pytest.warns` in `test_generate_percentiles.py`.